packages/openneuro-app/src/scripts/refactor_2021/redesign-form.jsx

```diff
@@ -0,0 +1,270 @@
+import React from 'react'
+import { Modal } from '@openneuro/components/modal'
+
+class ContactForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      modalities: [],
+      formCompleted: false,
+      showFeedbackModal: false,
+    }
+  }
+  myRequest = url => {
+    let response
+    try {
+      response = fetch(url, { method: 'POST' })
+    } catch (e) {
+      response = e
+    }
+    console.log(response)
```

Comment:
```suggestion
    fetch(url, { method: 'POST' })
```
Fetch returns a promise so this never throws.